js/core/config.d.ts

```diff
@@ -146,4 +147,10 @@ export interface globalConfig {
     useLegacyVisibleIndex?: boolean;
 }
 
+/**
+ * @docid
```

Comment:
This one doesn't need `docid` tag because it doesn't have it's own topic (`GlobalConfig` should have)
```suggestion
```